Changed About Section Text

cringeyburger opened the pull request:
Changed the text in AboutSection.astro

harshkhandeparkar commented on src/components/AboutSection.astro:
```diff
@@ -11,10 +11,8 @@ import Section from "./Section.astro";
-      Development Workflow, practice Algorithms and Data Structures, besides conducting workshops on GUI programming,
-      Web development, Android application Development and Hackathons.
+	A community of open-source enthusiasts at IIT Kharagpur, promoting open-source and FOSS through workshops, seminars, and events. Fostering a culture of technology and open source, we encourage learning, mentorship, and diverse software engineering projects.
+	With a focus on a love for coding, KOSS members pursue innovative ideas, from solving real-world problems to fun projects, while supporting each other's growth and expertise. Every year, we hold events to familiarise students with UNIX tools, the Linux environment, Git Development Workflow, algorithms, and data structures.
```
You will have to add a `<br />` element to introduce the line break. A normal newline does not work.
![image](https://github.com/kossiitkgp/YAKW/assets/34770591/1cb45aab-c6a6-4e46-85ce-e65f61e393b4)

cringeyburger replied:
Please review the changes

harshkhandeparkar commented on src/components/AboutSection.astro:
```diff
@@ -11,10 +11,10 @@ import Section from "./Section.astro";
-      Web development, Android application Development and Hackathons.
+	A community of open-source enthusiasts at IIT Kharagpur, promoting open-source and FOSS through workshops, seminars, and events.<br /> 
+	Fostering a culture of technology and open source, we encourage learning, mentorship, and diverse software engineering projects.<br />
+	With a focus on a love for coding, KOSS members pursue innovative ideas, from solving real-world problems to fun projects, while supporting each other's growth and expertise.<br /> 
```
I think it would look better if it is split into 2 paragraphs instead of 4.

proffapt reviewed (changes requested):
- Repetitive content like about events, promoting open-source etc. Would be better to not repeat.
- Also doing a center align might look good, idk, try that.

cringeyburger commented:
Please check the following changes: Removed repeating text, center aligned, added a new line in between the two paragraphs

chirag-ghosh commented:
![image](https://github.com/kossiitkgp/YAKW/assets/75582834/d8f77757-a260-45e4-a054-6bc762b7a745)

1. Center align looks bad here. The only place it will look good I think is in the mobile view. Reason: In desktop view the KOSS icon is on the left due to which left alignment goes with it. In mobile view, the icon is in the center which motivates for a center aligned text.
2. Instead of adding `<br/>` tags, you can add the two paragraphs in two different `<p>` tags (p here stands for paragraph) and add a margin bottom.

cringeyburger commented:
Please review the changes: Changed the text into two paragraphs. Added a bottom margin of 1rem. Changed alignment to the left.

Also, do I need to do something so that it changes the alignment to centre for mobile but stays on left when viewed on desktop?

Thanks

chirag-ghosh commented:
You can use media queries. If you add the center align style inside `@container Section (max-width: 768px)`` then it will only apply to devices with lower pixel width (generally mobile phones).

chirag-ghosh commented:
Great. this should work for now. About the other issue, just noticed the problem. I will be creating an issue addressing it. You can try fixing it. Ask for help if needed.

chirag-ghosh commented on src/components/AboutSection.astro:
```diff
@@ -68,6 +70,7 @@ import Section from "./Section.astro";
     }
 
     .about-description {
+      text-align: center; /* Added center alignment for smaller devices like mobile phones*/
```
This comment seems redundant since it is pretty evident from the code that this style is for smaller width devices.

cringeyburger replied:
Removed the comment
